src/install/install.zig

```diff
@@ -1789,6 +1795,26 @@ pub const PackageManager = struct {
         return this.allocator.create(NetworkTask) catch @panic("Memory allocation failure creating NetworkTask!");
     }
 
+    pub fn cachedGitHubFolderNamePrint(this: *const PackageManager, buf: []u8, name: string, repository: Repository) stringZ {
+        if (repository.committish.isEmpty()) {
+            return std.fmt.bufPrintZ(buf, "{s}@{s}!{s}", .{
```

Comment:
can we use a prefix instead of a special character? eg `gh@@`

https://stackoverflow.com/questions/36336518/issue-with-special-characters-in-path-exclamation-point-carrot-etc-using

Reply:
I can just reuse `@` instead of `!` since it won't clash with `semver` either way.